packages/extension-polkagate/src/fullscreen/stake/solo/commonTasks/configurePayee/index.tsx

```
@@ -31,13 +31,17 @@ interface Props {
   const theme = useTheme();
   const { t } = useTranslation();
 
-  const closeProxy = useCallback(
-    () => setStep && setStep(STEPS.REVIEW)
-    , [setStep]);
+  const _closeProxy = useCallback(() => {
+    if (closeProxy) {
+      return closeProxy();
+    }
+
+    setStep && setStep(STEPS.REVIEW);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)